doctr/models/detection/zoo.py

```diff
@@ -8,10 +8,19 @@
 from .. import detection
 
 
-__all__ = ["db_resnet50_predictor"]
+__all__ = ["db_resnet50_predictor", "db_resnet50_predictor_normed"]
 
 default_cfgs: Dict[str, Dict[str, Any]] = {
-    'db_resnet50_predictor': {'model': 'db_resnet50', 'post_processor': 'DBPostProcessor'},
+    'db_resnet50_predictor_normed': {
+        'model': 'db_resnet50',
+        'post_processor': 'DBPostProcessor',
+        'mean': (0.798, 0.785, 0.772),
+        'std': (0.264, 0.274, 0.287),
+    },
+    'db_resnet50_predictor': {
+        'model': 'db_resnet50',
+        'post_processor': 'DBPostProcessor',
+    },
```

Comment:
Actually, for now we have no "db_resnet50_predictor_normed" so I suggest:
- we just specify the old mean & std for "db_resnet50" in `differentiable_binarization.py`
- once we have a new (supposedly better) pretrained db_resnet50 with proper normalization, we upload the checkpoint, edit the mean, std, and url. (long term we have no need for lesser versions of the same model)

Reply:
so you don't need to modify anything in detection zoo apart from: passing
`mean=_model.cfg['mean'], std=_model.cfg['std']` to the `DetectionPreProcessor` in `_predictor`